.github/workflows/ci.yml

```diff
@@ -111,3 +111,55 @@ jobs:
 
       - name: carthage build
         run: ./script/carthage.sh build --cache-builds --no-skip-current
+
+  deploy-github:
+    name: Deploy Github
+    runs-on: macOS-10.15
+    needs: [build-test, swiftpm, cocoapods, carthage]
+    if: startsWith(github.ref, 'refs/tags/')
+    steps:
+      - name: git checkout
+        uses: actions/checkout@v2
+
+      - name: create release
+        uses: softprops/action-gh-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          body: |
+            # Changes
+
+            - <!-- Insert changes here -->
```

Comment:
Some leftovers?

Reply:
No, this was already in the release workflow job 😄 

It's more for us to already have some kind of format. Going forward we could get fancier and point to a CHANGELOG file or calculate the diff and automate this